apps/bfd-pipeline/bfd-pipeline-rda-bridge/pom.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>gov.cms.bfd</groupId>
+        <artifactId>bfd-pipeline-parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>bfd-pipeline-rda-bridge</artifactId>
+    <packaging>jar</packaging>
+
+    <name>RDA Bridge</name>
+    <description>Utility project for creating RDA data</description>
+
+    <properties>
+        <maven.compiler.source>8</maven.compiler.source>
+        <maven.compiler.target>8</maven.compiler.target>
+
+        <snakeyml.version>1.29</snakeyml.version>
+        <commons-cli.version>1.4</commons-cli.version>
+
+        <bfd-shared-utils.version>1.0.0-SNAPSHOT</bfd-shared-utils.version>
+        <bfd-pipeline-rda-grpcs.version>1.0.0-SNAPSHOT</bfd-pipeline-rda-grpcs.version>
+    </properties>
+
+    <dependencies>
+        <!-- Logging -->
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+        </dependency>
+        <!-- Utility -->
+        <dependency>
+            <groupId>org.yaml</groupId>
+            <artifactId>snakeyaml</artifactId>
+            <version>${snakeyml.version}</version>
+        </dependency>
+        <!-- Common -->
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>commons-cli</groupId>
+            <artifactId>commons-cli</artifactId>
+            <version>${commons-cli.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
```

Comment:
looks like a few new dependencies to BFD - snakeyml, commons-cli, and commons-lang3. Have those gone through the process for new dependencies?

Reply:
commons-lang3 should already exist as a transitive dep, seems hapi is bringing it in atm.  I'm not sure how the process works for new deps, nor do I know if it matters that this is not a deployed application, it's only run locally.

Reply:
@StewGoin - there are a few new dependencies here that will show up in the next inventory after this is merged. Would you like us to prepare any documentation for these?

Reply:
@StewGoin  Just to reiterate as well, the new dependencies are being brought in by a module that **will not** be deployed to any of our environments.  This will be run on our local machines only.

Reply:
Please include them (just a reference to them being new) in the overall SIA for PACA changes -- the BFD team will want to make sure it's reflected similar to other dependencies that are part of our dev/build process that we should be aware of as software supply chain even if they aren't being deployed to the production environment themselves.

Reply:
@StewGoin I've added these to the SIA